packages/plugin-amplitude/src/androidMain/kotlin/ly.iterative.itly.amplitude/AmplitudeOptions.kt

```diff
@@ -3,5 +3,8 @@ package ly.iterative.itly.amplitude
 import android.content.Context
 
 actual open class AmplitudeOptions(
-    val androidContext: Context
+    val androidContext: Context,
+    var branch: String? = null,
```

Comment:
@qingzhuozhen Seeing as these are on the top-level maybe we should prefix them with `plan`. Or maybe just `planVersion` What do you think?

Reply:
@justin-fiedler yeah, I was thinking about it earlier as well, the only reason why I decided to use directly `branch` or `version` is the `branch` or `version` was used in `IterativelyOptions` directly and comments mentioned they are for tracking plan, and use without prefix for consistence. I can change it to have prefix and name it as `planVersion` if we don't need to align with `IterativelyOptions`, let me know your thoughts.